velox/expression/LambdaExpr.cpp

```diff
@@ -49,30 +49,48 @@ class ExprCallable : public Callable {
       const std::vector<VectorPtr>& args,
       const BufferPtr& elementToTopLevelRows,
       VectorPtr* result) override {
-    std::vector<VectorPtr> allVectors = args;
-    for (auto index = args.size(); index < capture_->childrenSize(); ++index) {
-      auto values = capture_->childAt(index);
-      if (wrapCapture) {
-        values = BaseVector::wrapInDictionary(
-            BufferPtr(nullptr), wrapCapture, rows.end(), values);
-      }
-      allVectors.push_back(values);
-    }
-    auto row = std::make_shared<RowVector>(
-        context->pool(),
-        capture_->type(),
-        BufferPtr(nullptr),
-        rows.end(),
-        std::move(allVectors));
-    EvalCtx lambdaCtx(context->execCtx(), context->exprSet(), row.get());
+    auto row = createRowVector(context, wrapCapture, args, rows.end());
+    EvalCtx lambdaCtx = createLambdaCtx(context, row, finalSelection);
     ScopedVarSetter throwOnError(
         lambdaCtx.mutableThrowOnError(), context->throwOnError());
+    body_->eval(rows, lambdaCtx, *result);
+    transformErrorVector(lambdaCtx, context, rows, elementToTopLevelRows);
+  }
+
+  void applyNoThrowError(
+      const SelectivityVector& rows,
+      const SelectivityVector& finalSelection,
+      const BufferPtr& wrapCapture,
+      EvalCtx* context,
+      const std::vector<VectorPtr>& args,
+      VectorPtr& errors,
```

Comment:
VectorPtr -> ErrorVectorPtr (sorry, I got it wrong in my earlier suggestion)

Reply:
OK, done.